zubhub_frontend/zubhub/src/views/home/Home.jsx

```diff
@@ -327,7 +380,37 @@ function Projects(props) {
       }}
     >
             <div className={classes.carouselContainer}>
-              <Carousel showThumbs={false} infiniteLoop={true} showStatus={false} onChange={handleSlideChange}>
+              <Carousel showThumbs={false} infiniteLoop={true} showStatus={false} onChange={handleSlideChange} swipeable={false} autoPlay={true}
```

Comment:
Can you tell me more on whats causing the scrolling issue and then how the custom arrows fix the problem?

Reply:
@yokwejuste @chaitanya-cloudknox I was looking at the code and I have a question. Don't you think adding `infiniteLoop`true` recreates the problem we are trying to fix?

Reply:
No it doesn't, it fixes instead. The carousel package was causing it and I had to get a walk around to skip the blocked scroll

Reply:
> Can you tell me more on whats causing the scrolling issue and then how the custom arrows fix the problem?

The scroll was due to the axis assigned for the scroll, so all the interaction with the carousel interface brings us to a scenario where we have all action on the carousel only linked to the axis direction.